apps/src/lib/node/ledger/shell/vote_extensions.rs

```diff
@@ -18,9 +18,20 @@ mod extend_votes {
             &mut self,
             _req: request::ExtendVote,
         ) -> response::ExtendVote {
+            let validator_addr = match self.mode.get_validator_address() {
```

Comment:
We should put an expect here I think. Only validators should receive this method call.